esports_data_from_rimble/rimble/rimble.go

```diff
@@ -50,6 +50,7 @@ type Metadata struct {
 
 type MatchData struct {
 	Metadata    Metadata `json:"metadata"`
+	Date        string   `json:"date"`
```

Comment:
This is actually included in the match data response structure and can be used instead of passing it what we expect the date to be